SPECS/libyang/libyang.spec

```diff
@@ -0,0 +1,103 @@
+# valgrind finds invalid writes in libcmocka on arm and power
+# see bug #1699304 for more information
+%ifarch %{arm} ppc64le
+%global run_valgrind_tests OFF
+%else
+%global run_valgrind_tests ON
+%endif
```

Comment:
We don't support 32-bit ARM and PowerPC architectures, so we can remove this option.

Reply:
Ah I hadn't realised `arm` was 32-bit arm, thanks!